andorigna/src/component/page/Navbar/Navbar.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+import { Link } from 'react-router-dom';
+import "./Navbar.css";
+import logo from "../image/Logo.png";
+import Menu from "../image/Menu.png";
+import Close from "../image/Close.png";
+
+
+const Navbar = () => {
+
+  const [isOpen, setIsOpen] = useState(false);
+  return (
+<nav className="navbar">
+  <div className='logo-container d-flex'>
+    <div className='d-flex'>
+    <div className="navbar-image-container">
+    <img src={logo} alt= "logo andorinha"/></div>
```

Comment:
I recommend using tailwind instead of boostrap, so we're all using the same technology stack.